requirements.txt

```diff
@@ -45,3 +45,7 @@ python-glanceclient
 python-keystoneclient
 python-novaclient
 python-blazarclient
+
+celery
```

Comment:
Is this the celery internal library, or just a client to celery? I think we would only need the client here.

Reply:
I assume you are referring to the kombu? It's a transport library for celery. I believe we need it. I spent some time trying to figure out a compatible version for something that was breaking while testing.